ui/react-components/components/AppointmentStatus/AppointmentStatus.jsx

```diff
@@ -0,0 +1,73 @@
+import Dropdown from "../Dropdown/Dropdown.jsx";
+import React, { useEffect, useState } from "react";
+import Tags from "../Tags/Tags.jsx";
+import PropTypes from "prop-types";
+import { injectIntl } from "react-intl";
+import {searchFeildOnChangeHandler,searchFeildOnRemoveHandler} from '../../helper';
+
+const appointmentStatusList = [
+  { value: "Scheduled", label: "Scheduled" },
+  { value: "CheckedIn", label: "CheckedIn" },
+  { value: "Completed", label: "Completed" },
+  { value: "Cancelled", label: "Cancelled" },
+  { value: "Missed", label: "Missed" }
+];
```

Comment:
Could this come from constants instead of hardcoding it here? Similar to `PROVIDER_RESPONSE` in `FilterWrapper.jsx`.

Reply:
ok. done

Reply:
Yes. Added to constants . thanks

Reply:
Thanks.